dist/css/pop-up.css

```diff
@@ -0,0 +1,506 @@
+.project-btn {
+    height: 4.8rem;
+    width: 12.6rem;
+    left: 0;
+    top: 1.2rem;
+    border-radius: 8px;
+    border: 0.1rem solid #6070ff;
+    padding: 1.2rem;
+    background: #fff;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.6rem;
+    line-height: 2.4rem;
+    color: #396df2;
+    cursor: pointer;
+  }
+  
+  .project-btn:hover {
+    background-color: #6070ff;
+    color: #fff;
+    border: 1px solid #fff;
+  }
+  
+  .project-btn:active {
+    background: #396df2;
+  }
+  
+  .btn--green {
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    margin-left: 20px;
+  }
+  
+  /* Work Section */
+  .work {
+    margin-inline: 2.4rem;
+    padding-bottom: 10rem;
+  }
+  
+  .card__container {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    gap: 3rem;
+    justify-content: center;
+    padding-top: 12rem;
+  }
```

Comment:
- [ ] It is highly required to stick to the Figma design as close as possible. This includes everything on the layout - font, color, margin, padding among others. Your popup design does not look like how it is on the Figma design. Kindly give a closer look into the Figma design and walk your design alongside the figma both on the deskotp and the mobile.

### Your Design Desktop
![your-project-popup-desktop](https://github.com/mohashyne/MSalyhu_portfolio/assets/26597432/63de1c09-7d3b-4977-9a32-745efb741874)

### Figma Design Desktop
![temp-project-card-desktop](https://github.com/mohashyne/MSalyhu_portfolio/assets/26597432/02dae0bf-fc3d-4f95-92e4-8f970789ff4d)


### Your Design Mobile
![your-popu-project-card-mobile](https://github.com/mohashyne/MSalyhu_portfolio/assets/26597432/cc9088ce-3b67-4337-82be-e6593914af58)

### Figma Design Mobile
![temp-popup-mobile](https://github.com/mohashyne/MSalyhu_portfolio/assets/26597432/d8544f67-4467-44ff-b655-b24611feeb79)

Reply:
@mohashyne I have gone through your newly added changes and you seem to have not implemented the required changes from your first reviewer, it is important that your portfolio project matches closely the given [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as this shows how much attention to details you pay and it is a good quality to have as a frontend developer :+1: 
- [ ] With regards to implementing the design structure your `popup.css` files seems to be making use of multiple and repetitive styling, you have used two media queries of `768px` which shouldn't be the case however to closely match the given design i would suggest reducing the width of the popup on the desktop version and also dynamically loading the content of each popup rather than using a single image for all popup like you have done in `line 55` of you `popup.js` file